### PR TITLE
hotfix: redirect to specific thanks page

### PR DESCRIPTION
**Pull request by Hamms**
Since the generic "thanks" doesn't exist

### CODE REVIEW

**Comment by epeach** [comment]
Don't know a lot about this area of code or the intricacies of the problem, but as Dotd, this seems good to merge.

File: dashboard/app/controllers/pd/workshop_daily_survey_controller.rb [hunk 1]
```diff
@@ -96,7 +96,7 @@ def new_post
       enrollment = Enrollment.find_by!(code: params[:enrollment_code])
       workshop = enrollment.workshop
 
-      return redirect_to :thanks if WorkshopDailySurvey.exists?(user: current_user, pd_workshop: workshop, day: 5)
+      return redirect_to :pd_workshop_survey_thanks if WorkshopDailySurvey.exists?(user: current_user, pd_workshop: workshop, day: 5)
```
Comment (aoby) [inline dashboard/app/controllers/pd/workshop_daily_survey_controller.rb:99]: Thanks for jumping in and fixing this @Hamms! Actually this is supposed to be the `thanks` action on this controller. It should be `redirect_to action: :thanks` like [here](https://github.com/code-dot-org/code-dot-org/blob/d9155e6156ab2d205b27c0fd05e8f522e2f14336/dashboard/app/controllers/pd/workshop_daily_survey_controller.rb#L70) but for now this `pd_workshop_survey_thanks` will be just fine.

I'll make a note to updated it when I return.

Comment (aoby) [reply dashboard/app/controllers/pd/workshop_daily_survey_controller.rb:99]: And, of course that is just another way of referencing the same route so never mind. This is exactly right :)

Comment (Hamms) [reply dashboard/app/controllers/pd/workshop_daily_survey_controller.rb:99]: ah, yeah `action: :thanks` is probably more semantically correct, but I was rushing

**Pull request merged**